### DISCUSSION TITLE
How to create pg_profile dashboards for multiple PostgreSQL clusters in one Grafana instance?

### DISCUSSION THREAD

**Discussion by pgloader** [zubkov-andrei/pg_profile#91] [Q&A]
We want to use one Grafana instance to host pg_profile dashboards for multiple PostgreSQL clusters.  Is there a way to do it other than using one pg_profile instance for multiple clusters?

**Comment by zubkov-andrei** [comment]
Hello! Grafana dashboards are using postgres database containing pg_profile repo as a data source. The only way is to add to grafana several databases as a datasources and switch them in the dashboard.
